packages/modules/manage-versions/src/components/CreateUpdateVersion/VersionModal.tsx

```diff
@@ -116,6 +116,7 @@ export const VersionModal = (props: VersionModalProps) => {
         <ModalButtonBar>
           <Button
             styleType="high-visibility"
+            data-iui-variant="high-visibility"
```

Comment:
I used the migration tool provided by itwin and it was suggested change to include this data tag, but removed it since its working fine without that tag